buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -103,13 +105,16 @@ public String toString() {
     private final Property<Boolean> bundledJdk;
     private final Property<Boolean> failIfUnavailable;
     private final Configuration extracted;
+    private Function<ElasticsearchDistribution, ElasticsearchDistribution> distributionFinalizer;
```

Comment:
Couldn't this just be `Action<Distribution>`? I don't see where we ever actually use the returned result of this function.

Reply:
fixed